File: preview/hosted_numbers/dependent_hosted_number_order/read-default/read-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

dependent_hosted_number_orders = @client.preview.hosted_numbers
  .authorization_documents('PXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .dependent_hosted_number_orders
  .list

dependent_hosted_number_orders.each do |record|
  puts record.sid
end

File: api/v2010/transcription/read-default/read-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var transcriptions = TranscriptionResource.Read(
            pathAccountSid: "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        foreach(var record in transcriptions)
        {
           Console.WriteLine(record.Sid);
        }
    }
}
